docs/annexes/SPDX-license-expressions.md

```diff
@@ -50,7 +56,7 @@ In the `tag:value` format, a license expression MUST be on a single line, and MU
 
 ## D.2 Case sensitivity <a name="D.2"></a>
 
-License expression operators (`AND`, `OR` and `WITH`) should be matched in a *case-sensitive* manner.
+License expression operators (`AND`, `and`, `OR`, `or`, `WITH` and `with`) should be matched in a *case-sensitive* manner. I.e., variations `And`, `oR` are not allowed.
```

Comment:
I don't think the final explanation is needed.

If we want to have it in, reword it to something like:

> manner, i.e., letters must be all upper case or all lower case.